test/e2e/ha/scale_core_contrail_services_test.go

```diff
@@ -317,6 +332,159 @@ func TestHACoreContrailServices(t *testing.T) {
 				}
 			})
 		})
+		t.Run("when manager resource is upgraded", func(t *testing.T) {
+			var replicas int32 = 1
+			instance := &contrail.Manager{}
+			err := f.Client.Get(context.TODO(), types.NamespacedName{Name: "cluster1", Namespace: namespace}, instance)
+			assert.NoError(t, err)
+
+			zkContainer := utils.GetContainerFromList("zookeeper", instance.Spec.Services.Zookeepers[0].Spec.ServiceConfiguration.Containers)
+			zkContainer.Image = "registry:5000/common-docker-third-party/contrail/zookeeper:" + targetVersionMap["zookeeper"]
+
+			rmqContainer := utils.GetContainerFromList("rabbitmq", instance.Spec.Services.Rabbitmq.Spec.ServiceConfiguration.Containers)
+			rmqContainer.Image = "registry:5000/common-docker-third-party/contrail/rabbitmq:" + targetVersionMap["rabbitmq"]
+
+			//csContainer := utils.GetContainerFromList("cassandra", instance.Spec.Services.Cassandras[0].Spec.ServiceConfiguration.Containers)
```

Comment:
Maybe add a comment why this is commented out (test fails or something like this)

Reply:
Done